added licence in README

prasanta352 opened the pull request.

sharathks84 commented:
2. Readme.md follow the rules and template of community, 
                 a. where the original repo information is shared
                 b. Snapshots of how it will look in harmony is shared. images from Harmony emulator / device
3. gradle.properties should not have sonar local properties
systemProp.sonar.host.url=http://localhost:9000
4. build.gradle at project level
modify sonar for sonarcloud
property "sonar.host.url", "http://localhost:9000"
follow this
https://sonarcloud.io/project/configuration?analysisMode=GitHubActions&id=applibgroup_UniversalPickerDialog
5. config.json
Device type can be made "default" to support all the device forms

prasanta352 commented:
> 1. No difference between default picker and multipicker

changed it
![image](https://user-images.githubusercontent.com/32667365/136817843-19281960-4745-42f1-8de4-e4923243e17d.png)

> 1. Background and can be made same as original library
>    ![image](https://user-images.githubusercontent.com/2935164/136798013-d77017fc-5391-4621-aebe-6996f1a6e524.png)

changed it
![image](https://user-images.githubusercontent.com/32667365/136818011-e95062ac-6344-45ea-bea2-e66328c94e62.png)


> 1. Readme.md follow the rules and template of community,
>    a. where the original repo information is shared
>    b. Snapshots of how it will look in harmony is shared. images from Harmony emulator / device
> 2. gradle.properties should not have sonar local properties
>    systemProp.sonar.host.url=http://localhost:9000
> 3. build.gradle at project level
>    modify sonar for sonarcloud
>    property "sonar.host.url", "http://localhost:9000"
>    follow this
>    https://sonarcloud.io/project/configuration?analysisMode=GitHubActions&id=applibgroup_UniversalPickerDialog
> 4. config.json
>    Device type can be made "default" to support all the device forms

1. modified the README.md 
2. remove the "systemProp.sonar.host.url" property from gradle.properties
3. modified the "sonar.host.url" to "https://sonarcloud.io/" let me know if any more change is necessary 
4. modified the library(universalpickerdialog) module's config.json device type to "default", but when I change the example apps config.json it fails to deploy on the emulator.

> 1. First push should be the base version from where the version is taken.

I didn't understand what base version are you taking about ?

sharathks84 commented:
Hi prashanth better we cancel this pull request and apply new one.
1. Push the android library first and then pull the harmonyOS library on top of that.
2. refer other libs, since the env is not setup here, there are some backend issues with this pull.
3. DO this and raise the pull request will merge.
